server/security/collections.js

```diff
@@ -41,7 +41,7 @@ export default function () {
   Security.defineMethod("ifHasSellerRole", {
     fetch: [],
     deny: function (type, arg, userId) {
-      const isDenied = Roles.userIsInRole(["admin", "owner", "createProduct"], Reaction.getSellerShopId(userId));
+      const isDenied = !Roles.userIsInRole(userId, "createProduct", Reaction.getSellerShopId(userId));
```

Comment:
So a seller role is anyone who can `createProduct` for a given shop? Would be nice to have a comment here explaining this a little bit.